tests/test_sycl_queue.py

```diff
@@ -1299,20 +1299,21 @@ def test_asarray(device_x, device_y):
     assert_sycl_queue_equal(y.sycl_queue, x.to_device(device_y).sycl_queue)
 
 
+@pytest.mark.parametrize("func", ["take", "take_along_axis"])
 @pytest.mark.parametrize(
     "device",
     valid_devices,
     ids=[device.filter_string for device in valid_devices],
 )
-def test_take(device):
+def test_take(func, device):
```

Comment:
Do you want to add similar tests for `put` and `put_along_axis` here and in `test_usm_type.py`?

Reply:
No, because in case of `put` and `put_along_axis` we are not returning any array, but modifying input one.